client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button.jsx

```diff
@@ -58,21 +58,20 @@ export default React.createClass( {
 		} );
 
 		let { text } = this.props;
-		let buttonChildren;
 
 		if ( ! text ) {
 			text = this.translate( 'Disconnect', {
 				context: 'Jetpack: Action user takes to disconnect Jetpack site from .com'
 			} );
 		}
 
-		buttonChildren = (
+		const buttonChildren = (
 			<div>
 				{ text }
 				<DisconnectJetpackDialog site={ site } ref="dialog" redirect={ redirect } />
 			</div>
 		);
 
-		return React.createElement( buttonElement, buttonProps, buttonChildren );
+		return React.createElement( Button, buttonProps, buttonChildren );
```

Comment:
This breaks the intention of linkDisplay. A button component will render as a link, but requires the href prop to be defined.